### PR TITLE
Update sample .yaml to give noobs (like me) an easier starting point

### PR DESCRIPTION
**Pull request by GizmoGoody**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @GitHubGoody, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#22629).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/systemmonitor.markdown [hunk 1]
```diff
@@ -18,13 +18,43 @@ To add this platform to your installation,
 add the following to your `configuration.yaml` file:
 
 ```yaml
-# Example configuration.yaml entry
-sensor:
+# Example configuration.yaml entry with all entry types (delete/comment out as necessary)
+- sensor:
```
Comment (frenck) [inline source/_integrations/systemmonitor.markdown]: This line is actually incorrect now.

File: source/_integrations/systemmonitor.markdown [hunk 2]
```diff
@@ -18,13 +18,43 @@ To add this platform to your installation,
...
   - platform: systemmonitor
     resources:
       - type: disk_use_percent
         arg: /config
+      - type: disk_use
```
Comment (frenck) [inline source/_integrations/systemmonitor.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by GizmoGoody — COMMENTED** [review]
Made changes as requested by frenck

File: source/_integrations/systemmonitor.markdown [hunk 3]
```diff
@@ -141,3 +141,46 @@ Wireless LAN adapter Wireless Network Connection:
...
+      - type: packets_out
+        arg: eth0
+      - type: ipv4_address
+        arg: eth0
+      - type: ipv6_address
+        arg: eth0
+      - type: processor_use
+      - type: processor_temperature
+      # - type: process
+        # specify arg:
```
Comment (frenck) [inline source/_integrations/systemmonitor.markdown]: Commented out code, please remove that.

**Review by GizmoGoody — COMMENTED** [review]
Removed commented out code

**Review by frenck — APPROVED** [review]
Thanks, @GitHubGoody 👍

**Pull request merged**